compare_mt/compare_mt_main.py

```diff
@@ -364,6 +367,44 @@ def generate_sentence_examples(ref, outs, src=None,
   reporter.generate_report()
   return reporter 
 
+def generate_lang_id_report(ref, outs, model,
+                            min_length=5,
+                            print_lines=False,
+                            print_line_numbers=False):
+    wtl = WhatTheLang()
```

Comment:
You could make this conditional; only create the object when wtl is the lang id method.

Reply:
done